src/tools/yaml-viewer/index.ts

```diff
@@ -0,0 +1,12 @@
+import { AlignJustified } from '@vicons/tabler';
+import { defineTool } from '../tool';
+
+export const tool = defineTool({
+  name: 'YAML prettify and format',
+  path: '/yaml-prettify',
+  description: 'Prettify your YAML string to a human friendly readable format.',
+  keywords: ['yaml', 'viewer', 'prettify', 'format'],
+  component: () => import('./yaml-viewer.vue'),
+  icon: AlignJustified,
+  redirectFrom: ['/yaml-viewer'],
```

Comment:
Redirection is used for legacy routing support 
+ add the creation date 
```suggestion
  createdAt: new Date('2023-11-29'),
```